source/about-mongosync.txt

```diff
@@ -162,13 +162,15 @@ sync job.
 Considerations for Continuous Sync
 ----------------------------------
 
-Avoid using ``mongosync`` for disaster recovery for the following reasons:
+You can periodically run ``mongosync`` against source and destination clusters 
+in continuous synchronization. Using ``mongosync`` for continuous sync has the 
+following considerations:
 
 - For the destination cluster to have an up to date, consistent view of the 
   source cluster’s data, ``mongosync`` must :ref:`commit <c2c-api-commit>` . 
-  If a disaster occurs and shuts down the source cluster before ``mongosync`` 
-  commits, the destination cluster won't have a consistent snapshot of the 
-  source data and transactions may be torn. 
+  If the source cluster shuts down before ``mongosync`` can commit, such as in 
+  a disaster scenario, the destination cluster won't have a consistent snapshot 
+  of the source data and transactions may be torn.
```

Comment:
to "may be torn", we can add ", i.e., not be applied atomically", in case users haven't heard 'torn transactions' before

Reply:
It's against our documentation style to use abbreviations like `i.e.` since it's hard for international audiences to parse. Will opt for similar verbiage as in the [mongosync behavior page](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/mongosync/mongosync-behavior/#consistency:~:text=Transactions%20may%20appear%20non%2Datomically%20on%20the%20destination%20cluster):

> Transactions may appear non-atomically on the destination cluster